snippets/echo-terminal.mdx

```diff
@@ -1,9 +1,12 @@
-{/* Allow 1 tick for the terminal to render in the DOM before loading the terminal. */}
 export const EchoTerminal = () => {
-  setTimeout(() => {
-    if (typeof document !== "undefined") {
-      loadTerminal(); 
+  if (typeof document === "undefined") {
+    return null;
+  } else {
+    const clientComponent = document.getElementById("nv-echo-terminal-snippet");
+    if (clientComponent) {
+      clientComponent.innerHTML = '<nv-echo-terminal></nv-echo-terminal>';
     }
-  }, 1);
-  return <div id="nv-terminal" class="nv-code nv-terminal"></div>
+
+    return <div id="nv-echo-terminal-snippet"></div>
```

Comment:
❓ Can we `return <nv-echo-terminal></nv-echo-terminal>` and avoid a wrapping div or is MDX blocking us from doing so?

Reply:
Yep, I tried this but Mintlify's MDX compiler didn't like to use the web component directly.